app/views/assessor_interface/assessment_sections/show.html.erb

```diff
@@ -46,11 +46,20 @@
   <% end %>
 <% end %>
 
-<% if @assessment_section_view_object.show_online_checker? %>
-  <h2 class="govuk-heading-s">Online checker</h2>
-  <p class="govuk-body">This authority has an online checker for validating the supplied reference number:</p>
-  <p class="govuk-body"><%= govuk_link_to @assessment_section_view_object.online_checker_url, @assessment_section_view_object.online_checker_url, target: "_blank" %></p>
-<% end %>
+<%= govuk_accordion do |accordion| %>
```

Comment:
Do we need to put this within a conditional to make sure we don't render an empty accordion if the two accordion sections inside are not rendered? I wonder if the page looks like a strange with an empty accordion or if the accordion component handles that for us?